src/Microsoft.IdentityModel.Clients.ActiveDirectory/Core/Http/RedirectUriHelper.cs

```diff
@@ -55,7 +55,7 @@ public static void Validate(Uri redirectUri, bool usesSystemBrowser = false)
                     nameof(redirectUri));
             }
 
-            // Currenlty only MSAL supports the system browser, on Android and iOS
+            // Currently only MSAL supports the system browser, on Android and iOS
```

Comment:
>MSAL [](start = 30, length = 4)

not this PR: is this code then still relevant as it's not in ADAL? Is this dead code?